### DISCUSSION TITLE
Extra nodeset generated from subdomain subtracted sidesets

### DISCUSSION THREAD

**Discussion by BoZeng1997** [idaholab/moose#22311] [Q&A Meshing] [answered]
HI,

Is there a way to generate nodeset consists of all the nodes in a subdomain but not on the selected boundaries?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Not in the current set of mesh generators. This could be a custom one you make for your application though, it would not be very difficult to code in C++ in MOOSE

Guillaume

**Reply by friedmud** [reply]
And if you do make it - it would be great if you want to contribute it back to MOOSE by putting up a PR.

**Reply by BoZeng1997** [reply]
I will try my best!
